src/site/content/en/blog/http-cache-security/index.md

```diff
@@ -0,0 +1,177 @@
+  - arthursonzogni
+description: Forgetting or misusing the Cache-Control header might negatively impact the security of your website and your users' privacy. Get recommendations for high-value websites.
+date: 2022-02-07
+thumbnail: image/VbsHyyQopiec0718rMq2kTE1hke2/3MnHBfutFgu2ai1F2mUH.jpg
+alt: Two ducks move through the water, creating a rippling wave.
+tags:
+  - blog
+  - security
+---
+
+By default, resources are always allowed to be cached by any type of cache.
+Not using or misusing the `Cache-Control` header might negatively impact the
+security of your website and your users' privacy.
+
+For personalized responses you want to keep private, we recommend you either:
+
+*  Set an appropriate [secondary cache
+   key](https://httpwg.org/specs/rfc7234.html#caching.negotiated.responses).
+   If the response depends on cookies, then set `Vary: Cookie`.
+*  Prevent intermediaries from caching the resource. Set
+   `Cache-Control: private`.
+
+Read on to learn why this matters and discover:
+
+1. Security and privacy problems you might be unaware of
+2. Different types of HTTP caches and common misconceptions
+3. Recommended actions for high-value websites
+
+## Cache-related security and privacy risks
+
+### Leaky resources from Spectre vulnerabilities
+
+The [Spectre
+vulnerability](https://en.wikipedia.org/wiki/Spectre_(security_vulnerability)) allows a page to read an OS processor's memory. This means an attacker can
+gain unauthorized access to cross-origin data. As a consequence, modern web
+browsers have restricted usage of some of their features&mdash;such as
+[`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+or [high resolution timer](https://developer.chrome.com/blog/cross-origin-isolated-hr-timers/)&mdash;to
+pages with [cross-origin
+isolation](/cross-origin-isolation-guide/). 
+
+Modern web browsers enforce [Cross-Origin Embedder Policy
+(COEP)](https://developer.chrome.com/blog/coep-credentialless-origin-trial/). This ensures cross-origin
+resources are either:
+
+*  Public resources, requested without cookies
+*  Resources explicitly allowed to be shared cross-origin, via CORS or the
+   [CORP](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP))
+   header
+
+This setup doesn't prevent an attacker from exploiting Spectre. However, it
+ensures cross-origin resources are not valuable to the attacker (= public
+resource) or allowed to be shared with the attacker (=
+`[CORP](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)):cross-origin`).
+
+### How does HTTP caching affect Spectre?
+
+If the `Cache-Control` header isn't properly set, an attacker could execute an
+attack. For example:
+
+1.  The credentialed resource is cached.
+1.  The attacker loads a cross-origin isolated page.
+1.  The attacker requests the resource again.
+1.  `COEP:credentialless` is set by the browser, so the resource is fetched
```

Comment:
Perhaps we can add a link to https://developer.chrome.com/blog/coep-credentialless-origin-trial/ ?